tests/integration/graviton_test.py

```diff
@@ -174,7 +169,7 @@ def test_stable_teal_generation(subr, mode):
         "inputs": [()],
         # since only a single input, just assert a constant in each case
         "assertions": {
-            DRProp.cost: 11,
+            DRProp.cost: lambda _, actual: actual in {11, 12},
```

Comment:
This change (and other similar) is to accommodate running both v6 and v8 (frame pointers) which result in slight differences.